sylius/multi-store-plugin/0.2/config/packages/sylius_multi_store_plugin.php

```diff
@@ -0,0 +1,9 @@
+<?php
+
+declare(strict_types=1);
+
+use Symfony\Component\DependencyInjection\Loader\Configurator\ContainerConfigurator;
+
+return static function (ContainerConfigurator $containerConfigurator): void {
+    $containerConfigurator->import(__DIR__.'/@SyliusMultiStorePlugin/config/config.php');
```

Comment:
The `__DIR__` seems to be incorrectly handled here: https://github.com/Sylius/SyliusRecipes/blob/7124e6dc10f663a206bdae816dfce9456f6bf773/sylius/multi-store-plugin/0.2/config/packages/sylius_multi_store_plugin.yaml#L1-L2